src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/FindCollectionsOperation.java

```diff
@@ -58,23 +69,60 @@ public Uni<Supplier<CommandResult>> execute(QueryExecutor queryExecutor) {
         .filter(tableMatcher)
 
         // map to name
-        .map(Schema.CqlTable::getName)
+        .map(table -> CollectionSettings.getVectorProperties(table, objectMapper))
 
         // get as list
         .collect()
         .asList()
 
         // wrap into command result
-        .map(Result::new);
+        .map(properties -> new Result(explain, properties));
   }
 
   // simple result wrapper
-  private record Result(List<String> collections) implements Supplier<CommandResult> {
+  private record Result(boolean explain, List<CollectionSettings> collections)
+      implements Supplier<CommandResult> {
 
     @Override
     public CommandResult get() {
-      Map<CommandStatus, Object> statuses = Map.of(CommandStatus.EXISTING_COLLECTIONS, collections);
-      return new CommandResult(statuses);
+      if (explain) {
+        final List<CreateCollectionCommand> createCollectionCommands =
```

Comment:
Add a comment explaining that "CreateCollectionCommand" created for convenience; we are not creating collections? (but want to expose same set of properties that we take when creating).